### PR TITLE
Feat/dao proposals filter nothing found

### PR DESCRIPTION
**Pull request by Zuckerwattederivat**
#264

### CODE REVIEW

File: assets/locales/en.json [hunk 1]
```diff
@@ -9,6 +9,17 @@
       "counterVotes": "No Votes",
       "requestedAmount": "Requested Amount",
       "votes": "Yes Votes"
+    },
+    "filterResponse": {
+      "noSearch": {
+        "heading": "No proposals <span class='text-primary'>yet :(</span>",
```
Comment (Abrom8) [inline assets/locales/en.json:15]: issue(blocking): Let's avoid using html and css classes in our locales.
suggestion: Move html into the component

File: assets/locales/en.json [hunk 2]
```diff
@@ -9,6 +9,17 @@
...
+        "paragraph": "Your chance to stand out as a project.",
+        "button": "Propose Now"
+      },
+      "search": {
+        "heading": "No proposal <span class='text-primary'>found :(</span>",
```
Comment (Abrom8) [inline assets/locales/en.json:20]: see above

File: components/app/dao-proposals/DaoProposalsList.vue [hunk 3]
```diff
@@ -1,102 +1,107 @@
...
-            <div class="mr-3">
-              <app-logo
-                class="inline-block"
-                :src="daoProposal.project.logo"
-                :alt="daoProposal.project.title"
-                :size="45"
-              />
+  <div class="mt-10">
+    <dao-proposals-response
+      :proposals-found="!!daoProposals.length"
```
Comment (Abrom8) [inline components/app/dao-proposals/DaoProposalsList.vue:4]: nitpick: !! can be removed

File: components/app/dao-proposals/DaoProposalsList.vue [hunk 4]
```diff
@@ -1,102 +1,107 @@
...
+      class="
+        grid grid-cols-1
+        md:grid-cols-2
+        lg:grid-cols-2
+        xl:grid-cols-4
+        2xl:grid-cols-4
+        gap-4
+      "
+    >
+      <div v-for="daoProposal in daoProposals" :key="daoProposal._id">
```
Comment (Abrom8) [inline components/app/dao-proposals/DaoProposalsList.vue:17]: suggestion(blocking): we use id instead of _id now

**Review by Abrom8 — CHANGES_REQUESTED** [review]

File: components/app/dao-proposals/DaoProposalsList.vue (hunk 3 above)
Comment (Zuckerwattederivat) [reply components/app/dao-proposals/DaoProposalsList.vue:4]: The prop requires a boolean type. Falsy or truthy values throw errors accordingly.

**Review by marhali — APPROVED** [review]

**Pull request merged**